modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/TpchQueryPlannerTest.java

```diff
@@ -37,13 +37,6 @@
 )
 public class TpchQueryPlannerTest extends AbstractTpcQueryPlannerTest {
     @ParameterizedTest
-    // TODO: https://issues.apache.org/jira/browse/IGNITE-24195
```

Comment:
Should it be rewritten to use the column names instead of $-named variable? It's good point to understand the issue and  example where we can came into it

Reply:
query 7 used to be commented out (i.e. disabled). In the #5177 I uncommented the query to update the expected plan, but forgot to disabled it back. Apparently, mentioned patch changed not only format of tree, but also the way condition is printed out (SEARCH operator is no longer expanded before printing out for explain). This makes original comment irrelevant in this particular case, therefore I deleted it

Reply:
Got it. In this case, the IGNITE-24195 can potentially be closed too.